.github/workflows/build.yml

```diff
@@ -3,84 +3,149 @@ name: Build Docker Image
     permissions:
-      id-token: write
       contents: read
-      packages: write
     steps:
-      -
-        name: Get commit message
-        id: commit_message
-        uses: sergeysova/jq-action@v2
-        with:
-          cmd: echo -n "${{github.event.head_commit.message}}" | jq -Rsa . | sed -e 's/^"//' -e 's/"$//'
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-      -
-        name: Login to Docker Hub
-        uses: docker/login-action@v3
-        with:
-          username: crestal
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      -
-        name: Build and push
-        uses: docker/build-push-action@v5
-        with:
-          build-args: |
-            RELEASE=${{ github.run_number }}
-          push: true
-          tags: crestal/intentkit:latest,crestal/intentkit:${{ github.sha }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-      -
-        name: Post to a Slack channel
-        if: ${{ success() }}
-        id: slack-success
-        uses: slackapi/slack-github-action@v1.26.0
-        with:
-          channel-id: 'C0786MHAL8J'
-          payload: |
-            {
-              "blocks": [
-                {
-                  "type": "section",
-                  "text": {
-                    "type": "mrkdwn",
-                    "text": "`${{ github.repository }}`\n*Branch*: `${{ github.ref_name }}`\n*Build result*: (${{ github.run_number }}) *${{ job.status }}* 🟩\n*Changes*:\n${{ steps.commit_message.outputs.value }}\n*Diff*: ${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
-                  }
-                }
-              ]
-            }
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      -
-        name: Post to a Slack channel
-        if: ${{ failure() }}
-        id: slack-failure
-        uses: slackapi/slack-github-action@v1.26.0
-        with:
-          channel-id: 'C0786MHAL8J'
-          payload: |
-            {
-              "blocks": [
-                {
-                  "type": "section",
-                  "text": {
-                    "type": "mrkdwn",
-                    "text": "`${{ github.repository }}`\n*Branch*: `${{ github.ref_name }}`\n*Build result*: (${{ github.run_number }}) *${{ job.status }}* 🟥\n*Build Log*: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}\n*Changes*:\n${{ steps.commit_message.outputs.value }}\n*Diff*: ${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
-                  }
-                }
-              ]
-            }
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+    - name: Build Start
+      id: ci_start
+      uses: slackapi/slack-github-action@485a9d42d3a73031f12ec201c457e2162c45d02d
+      with:
+        method: chat.postMessage
+        token: ${{ secrets.SLACK_BOT_TOKEN }}
+        payload: |
+          channel: 'C0786MHAL8J'
```

Comment:
this should be a config var

Reply:
maybe shouldnt be in `main`

Reply:
I'll add the configuration at the repository level. What does "maybe shouldn't be in main" mean?

Reply:
this is a centralized slack notification decision - it's not part of intentkit core build

Reply:
but saving at repo level is ok for now